### DISCUSSION TITLE
Hugo Theme for Docs site

### DISCUSSION THREAD

**Discussion by Prakash4844** [Prakash4844/tit#25] [Ideas]
@SakuraBlossomTree, I have few themes in mind.

1. https://themes.gohugo.io/themes/docsy/

2. https://themes.gohugo.io/themes/doks/

3. https://themes.gohugo.io/themes/lotusdocs/

4. https://themes.gohugo.io/themes/hugo-theme-bootstrap/

**Comment by SakuraBlossomTree** [comment]
I will be busy for a couple of days 

I will check after a few days

**Reply by Prakash4844** [reply]
No problem, i am working on CI part right now.

**Reply by SakuraBlossomTree** [reply]
Alright
